client/src/components/rapid/NextRapid.js

```diff
@@ -9,14 +9,14 @@ import GeneralButton from "../general/GeneralButton";
  * Button to navigate to a connected rapid in the river.
  */
 
-const NextRapid = ({ linkId, bottom, right, river }) => {
+const NextRapid = ({ linkId, bottom, right, section }) => {
```

Comment:
I am assuming this returns a button to the next section of the river. 
May be change naming to `NextSection`?